packages/manager/modules/pci/src/projects/project/storages/blocks/blocks.service.js

```diff
@@ -48,7 +51,12 @@ export default class PciProjectStorageBlockService {
         const instanceIds = uniq(
           reduce(
             volumes,
-            (instanceAcc, volume) => [...instanceAcc, ...volume.attachedTo],
+            (instanceAcc, volume) => {
+              if(isArray(volume.attachedTo) && !isEmpty(head(volume.attachedTo))) {
```

Comment:
is it ok that you are only checking that the first element of volume.attachedTo is empty ?

Reply:
yes, the volume can only be attached to one instance. I verified it. Manager doe not allows attaching a volume to multiple instances.